generation/WinSDK/enums.json

```diff
@@ -3861,6 +3861,7 @@
     "type": "uint",
     "namespace": "Windows.Win32.Graphics.Gdi",
     "name": "TEXT_ALIGN_OPTIONS",
```

Comment:
Should `TEXT_ALIGN_OPTIONS` use `autoPopulate`? Would that mean we would only need to define `TA_MASK` in `members`?

Reply:
Yes, you could do that, but I would only change it if something is broken. If it's already working, I would leave it as is.

Reply:
I will leave it as is then.